docs/DeveloperGuide.md

```diff
@@ -495,6 +495,33 @@ testers are expected to do more *exploratory* testing.
    1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
       - Expected: Similar to previous.
 
+### Command history
+1. Navigating through command history with the up/down arrow keys
+
+    1. Prerequisites: Enter several different commands (e.g., listemployees, addproject ProjectX, delete 1), so that there is a history of commands available.
```

Comment:
use code typeface for the commands

use `addproject pn/Project X pid/3` for the addproject command so testers can actually use it

Reply:
okay 
editing it now